server/database/schema.sql

```diff
@@ -48,16 +48,16 @@ CREATE TABLE menu (
 CREATE TABLE recipe (
     id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
     name VARCHAR(255) NOT NULL,
-    number_of_people INT,
+    number_of_people INT UNSIGNED NOT NULL,
     description TEXT NOT NULL,
     image TEXT NOT NULL,
-    date DATE NOT NULL,
-    is_favorite BOOLEAN,
-    vote INT,
-    set_up_time INT,
-    is_validated BOOLEAN,
-    user_id INT NOT NULL,
-    badge_id INT NOT NULL,
+    date DATE DEFAULT "1900-01-01",
```

Comment:
prefer a "DEFAULT NOW()" here

Reply:
I changed it to a DATETIME DEFAULT NOW()